Avoid an implicit declaration of malloc.  This is a real bug: The
implied int return type results in pointer truncation, so this code
cannot work.

diff --git a/inetcf.c b/inetcf.c
index d77236782b53d47a..007b95afc96a179c 100644
--- a/inetcf.c
+++ b/inetcf.c
@@ -14,6 +14,7 @@ static char sccsid[] = "@(#) inetcf.c 1.7 97/02/12 02:13:23";
 #include <stdio.h>
 #include <errno.h>
 #include <string.h>
+#include <stdlib.h>
 
 extern int errno;
 extern void exit();
